### ISSUE TITLE
Conclusion / Final wrap-up

### ISSUE THREAD

**Issue by lesteve** [INRIA/scikit-learn-mooc#261]
One short page:
- pointer to scikit-learn doc
- how the doc is organised
  + user guide
  + API
  + examples
- doc scikit-learn
- where to ask questions

**Comment by lesteve** [comment]
Look at Andreas Mueller books Wrapping up as inspiration. He talks about
machine learning as a small part of the problem most of the times. so how you
approach this, how you put it in production.

Things to go further : books, advanced machine learning Andreas Mueller, Jake
van der Plas Python for Data Analysis, Kaggle.

Scikit-learn documentation. How it is structured (user guide, examples, API).
Where to ask for help. How to contribute. Mention github open-source guides.

Things we have not seen: unsupervised learning, deep learning (probably not ...), reinforcement learning (probably not ...)

**Comment by GaelVaroquaux** [comment]
Notes:

- We gave methodological elements, but these are not enough to always have solid conclusion
- Say that machine learning is only a small part of a bigger picture
- Mention biases in the data, choice of the output, how the prediction is used.
- Congratulate people for getting there, Thank everyone (students, instructors, people who helped on the forum)

Possible inspiration: https://lms.fun-mooc.fr/courses/course-v1:inria+41016+self-paced/jump_to_id/e856f7062d334dfa9d02f621825c1d00

**Comment by ogrisel** [comment]
We could also link to resources such as https://fairlearn.org/ as a practical intro to some problems caused by a too naive machine application.

**Comment by ArturoAmorQ** [comment]
I would propose to do a video containing the points mentioned above by Gael plus the "real life cost of decision making based on ML" e.g. MAE in economics, and false positives/negatives in health data. That would bring back the idea of applications to the real world and the absolute need for doing it properly. This feels like priority for version 2.0.

The "Things to go further" proposal could also be done for this session, but as a notebook containing external links, instead of a video.

Finally, a shorter (separated) video to mention some of the things we did not cover (unsupervised learning, deep learning, reinforcement learning, etc) as proposed by @lesteve feels also important but could be done for version 3.0.

What do you think?

**Comment by ogrisel** [comment]
Suggestions of points to address:

- Consider the impacts of bugs in code used to extract the input features;
- Consider the impact of of the sampling procedure used to build the training set compared to the data that will be observed for later predictions.
- Consider the potential lack of symmetry in the cost (money, impact on users, human lives, ...) of making different kinds of predictions errors (e.g. false positive vs false negatives) and chose or build a model selection metric accordingly.
- Is machine learning really the right tool to try to tackle a given problem? Sometimes a hard-coded expert rule can be almost as accurate, much easier to debug, deploy and maintain overtime, easier to introspect and explain to the users or regulators.
- Some kinds of classifications (e.g. classify the gender of users based on a photo or attributes) are sometimes more harmful than useful and building systems to make those kinds of sensitive automated judgements should rather be avoided completely.

**Comment by ogrisel** [comment]
> What do you think?

I would start with a short video on a markdown page with additional info (not necessarily covered in the video) and links to additional references.

If this videos or the page becomes too long we can later split it in more topical pages in `MOOC 3.0`.

**Comment by MarieCollin** [comment]
Suggestion for the Conclusion Module organisation in FUN
<img width="747" alt="Capture d’écran 2022-01-25 à 14 12 57" src="https://user-images.githubusercontent.com/59566346/151023525-488e7907-39dc-4971-b113-17f90028d7bd.png">

**Comment by ArturoAmorQ** [comment]
> Suggestion for the Conclusion Module organisation in FUN

The placing LGTM once the video and slides are merged.

**Comment by MarieCollin** [comment]
Gael planned to use the slides only as a guide to prepare the talk. The idea was to produce a video without slides but with added bullet points or images to guide the learners. 
A first version of the video with integrated slides (produced by the shooting team in order to help us define encrusted texts and images) is available on mybox : https://mybox.inria.fr/f/eba14c2d3faa4a62b873/ 

As I am about to to start working on the encrusted texts and images on the video, it's better to know as soon as possible.
What do you think?

**Comment by ArturoAmorQ** [comment]
> Gael planned to use the slides only as a guide to prepare the talk. The idea was to produce a video without slides but with added bullet points or images to guide the learners.

I guess the rendering on FUN made me think we were using the slides

![screenshot](https://user-images.githubusercontent.com/86408019/151170873-6f9e0c09-85d2-49e3-a0b9-9d1a7a4da7fd.png)

> What do you think?

The video is good as it is. If we can synchronize the slides with the rendering of #541 to avoid having typos would be even better. 

In terms of images I think we don't have enough time or great need for adding them. We can keep the current version.

**Comment by MarieCollin** [comment]
A compromise could be to keep the technical parts of the slides and add a few icon images at the beginning and end of talk, where texts have no added value.
Of course the typos will be fixed. We have to do it manually on the video: there is no automatic synchronization between the text and the video as the slides are video-captured during the shooting.
